### DISCUSSION TITLE
VM dashboard  has no data

### DISCUSSION THREAD

**Discussion by renhaizhou** [apache/skywalking#7639] [Q&A] [answered]

**Comment by wu-sheng** [comment]
Have you read set up document? Your question doesn't include any information.

**Reply by renhaizhou** [reply]
![image](https://user-images.githubusercontent.com/41145169/131934115-6900e87b-090b-4e3c-93dc-c8892dadbce0.png)
It doesn't work to restart on this setting

**Comment by wankai123** [comment, marked as answer]
Please read this doc: https://skywalking.apache.org/docs/main/latest/en/setup/backend/backend-vm-monitoring/
It needs to configure `receiver-otel` not `agent-analyzer`

**Comment by ghost** [comment]
I am following the below for Infrastructure monitoring of my server.
https://skywalking.apache.org/blog/2021-02-07-infrastructure-monitoring/ 

When i am about to start the docker container, I get the following error

otel-collector_1  | 2022-01-23T06:33:36.319Z    warn    internal/metricsbuilder.go:104  Failed to scrape Prometheus endpoint    {"component_kind": "receiver", "component_type": "prometheus", "component_name": "prometheus", "scrape_timestamp": 1642919616318, "target_labels": "map[instance:127.0.0.1:9100 job:vm-monitoring]"}

Please guide how to counter the above error

**Reply by wu-sheng** [reply]
Please ask the OpenTelemetry community. This is out of SkyWalking's scope.

**Reply by ghost** [reply]
I have the same error with the "SelfObservability" dashboard that no data is showing there too. Need some guidance regarding that too please

**Reply by wu-sheng** [reply]
https://skywalking.apache.org/docs/skywalking-showcase/latest/readme/

Showcase has locked all versions to deploy everything out of the box. But OpenTelemtry collector is really not built by the SkyWalking community.

**Reply by ghost** [reply]
Thanks for the reply. I'll get back for queries related to skywalking
